rasa/train.py

```diff
@@ -477,6 +480,7 @@ async def _train_nlu_async(
     fixed_model_name: Optional[Text] = None,
     persist_nlu_training_data: bool = False,
     additional_arguments: Optional[Dict] = None,
+    domain: Optional[Union[Domain, Text]] = None,
 ):
```

Comment:
this is where it's supposed to go :-D
```suggestion
) -> Optional[Text]:
```